scheduler/kubernetes/csi.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: page
+title: "Deploying Portworx with CSI support"
+keywords: csi, portworx, container, Kubernetes, storage, Docker, k8s, pv, persistent disk 
+sidebar: home_sidebar
+---
+
+* TOC
+{:toc}
+
+[CSI](https://kubernetes-csi.github.io/), or _Container Storage Interface_, is
```

Comment:
should we mention that CSI is in beta in K8s?

Reply:
Yeah, I'll add that.

Reply:
done